substrafl/strategies/strategy.py

```diff
@@ -44,8 +43,8 @@ def initialization_round(
             # define composite tasks (do not submit yet)
```

Comment:
are composite tasks still a valid terminology?

Reply:
Nop, maybe this worth an other PR, I'll create a card if needed, thanks :)